test/conformance/tests/sriov_operator.go

```diff
@@ -936,6 +944,50 @@ var _ = Describe("[sriov] operator", func() {
 			})
 		})
 
+		Context("NAD update", func() {
+			// 24713
+			It("NAD is updated when SriovNetwork spec/networkNamespace is changed", func() {
+				ns1 := "z1"
```

Comment:
Can you name the namespace as `test-` so it's more clear they are test related?
Also, I would use `namespaces.Test` for at least one of the two, so it's collected by `namespaces.Clean` in case something goes wrong.